src/main/java/controller/ArticleController.java

```diff
@@ -13,22 +13,44 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 import javax.servlet.http.HttpSession;
+import java.io.IOException;
 import java.util.List;
 
 public class ArticleController extends HttpServlet {
 
-    @Override
-    public void doGet(HttpServletRequest req, HttpServletResponse resp)
-            throws ServletException, java.io.IOException {
+    IArticleService articleService = new ArticleService();
 
-        IArticleService articles = new ArticleService();
+    @Override
+    public void doGet(HttpServletRequest req, HttpServletResponse resp) {
 
         String id = req.getParameter("id");
 
-        ArticleDTO articleDTO = articles.getArticle(id);
-        req.setAttribute("articleDTO", articleDTO);
-        TopicDTO topicDTO = new TopicDTO("","");
-        req.setAttribute("topicDTO", topicDTO);
+        ArticleDTO articleDTO = articleService.getArticle(id);
+
```

Comment:
Sorry, correct me if I am wrong, but why do we need those empty spaces?

Reply:
empty spaces deleted